.eslintrc.json

```diff
@@ -3,8 +3,14 @@
     "eslint-config-airbnb",
     "next/core-web-vitals"
   ],
+  "env": {
+    "jest": true
+  },
   "rules": {
     // Other rules
-    "@next/next/no-img-element": "off"
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "@next/next/no-img-element": "off",
+    "jsx-a11y/anchor-is-valid": "off",
+    "@next/next/no-html-link-for-pages": "off"
```

Comment:
I'm going to picky here and say that the only two rules that we should be globally disabling are `next/no-image-element` and `jsx-a11y/anchor-is-valid`. I'd prefer if those other things were fixed in the code.

For `jsx-file-name-extension`, that would mean changing `index.test.js` to `index.test.jsx`, and I'm not sure what `no-html-link-for-pages` was warning about, but we should fix it wherever eslint is complaining about it.